lib/date.js

```diff
@@ -136,6 +136,10 @@ export function getDateArray(data, startDate, endDate, unit) {
 }
 
 export function getDateLength(startDate, endDate, unit) {
+  if (!unit) {
+    unit = 'day';
+  }
```

Comment:
this unit, somehow can be undefined.

this is bit of ugly, but this default value can help app running without unexpected stop.